_content/av/anger-its-worse-than-you-think_ada-kase.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Anger - It's Worse Than You Think and How to Manage It"
+authors:
+  - "Ada Kase"
+external_url: "https://www.amazon.com/The-Buddhist-Auntie-Podcast/dp/B08JJQWDVG"
```

Comment:
Yeah I believe she's taken the podcast offline, right? Not sure why. I hope not due to harassment :/  In such cases, I will move this link to `source_url` and I upload the file to the "exclusive content bucket" and add:

```yaml
file_links:
  - "exclusive_01/Anger_-_It's_Worse_Than_You_Think_and_How_to_Manage_It%20(2019-01-24)%20-%20Buddhist%20Auntie.mp3"
```